assets/aml-benchmark/components/batch_benchmark_inference_claude/spec.yaml

```diff
@@ -4,7 +4,7 @@ type: pipeline
 name: batch_benchmark_inference_claude
 display_name: Batch Benchmark Inference with claude support
 description: Components for batch endpoint inference
-version: 0.0.3
+version: 0.0.2
```

Comment:
Can we roll back this one? Claude component is not for release here.

Reply:
the live is 0.0.2, we updated this last time and hence downgraded it to match the one in prod registry.